docs/source/en/model_doc/layoutlmv3.md

```diff
@@ -14,24 +14,150 @@ rendered properly in your Markdown viewer.
 
 -->
 
+[![PyTorch](https://img.shields.io/badge/PyTorch-1.12+-blue.svg)](https://pytorch.org/get-started/locally/)
```

Comment:
Please style these with the `<div>` tags. You can copy it from one of the existing updated model cards on `main` like Gemma 3.